docs/monitor-template.md

```diff
@@ -0,0 +1,224 @@
+#monitor template
+
+###signalfx-agent repo/docs
+
+
+
+_INSTALL TAB_ (on top)
+
+\<monitor logo>
+# \<Monitor name> Installation
+
+Metadata associated with the \<name> collectd plugin can be found _here_. The relevant code for the plugin can be found _here_.
```

Comment:
We discussed moving this paragraph farther down. Seems like it would best fit under the Description heading.

Reply:
this comment is on the monitor-template file that has gotten in the PR by mistake. I am going to delete it from this push if I can.